### ISSUE TITLE
Make Pattern: Deprecated Revit 2014 API

### ISSUE THREAD

**Issue by yagbugs** [eirannejad/pyRevit#510]
I'm running revit 2014 and I'm having a pop up with lines of issue when I click the "Make Pattern" button. I've uninstalled and reinstall pyrevit and the same thing keep popping up when I click the "Make Pattern" whether drawings are highlighted or not, below is the pop up message I've been seeing. Can you please help with this issue

```
IronPython Traceback:
Traceback (most recent call last):
 File "C:\Users\Dacosta\AppData\Roaming\pyRevit-Master\extensions\pyRevitTools.extension\pyRevit.tab\Modify.panel\Patterns.splitpushbutton\Make Pattern.pushbutton\script.py", line 391, in 
 File "C:\Users\Dacosta\AppData\Roaming\pyRevit-Master\extensions\pyRevitTools.extension\pyRevit.tab\Modify.panel\Patterns.splitpushbutton\Make Pattern.pushbutton\script.py", line 86, in __init__
 File "C:\Users\Dacosta\AppData\Roaming\pyRevit-Master\extensions\pyRevitTools.extension\pyRevit.tab\Modify.panel\Patterns.splitpushbutton\Make Pattern.pushbutton\script.py", line 214, in setup_view_scale
TypeError: expected str, got BuiltInParameter

Script Executor Traceback:
Microsoft.Scripting.ArgumentTypeException: expected str, got BuiltInParameter
 at Microsoft.Scripting.Interpreter.ThrowInstruction.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.LightLambda.Run4[T0,T1,T2,T3,TRet](T0 arg0, T1 arg1, T2 arg2, T3 arg3)
 at IronPython.Runtime.Types.BuiltinFunction.Call(CodeContext context, SiteLocalStorage`1 storage, Object instance, Object[] args)
 at IronPython.Runtime.Types.ReflectedGetterSetter.CallGetter(CodeContext context, SiteLocalStorage`1 storage, Object instance, Object[] args)
 at IronPython.Runtime.Types.ReflectedIndexer.get_Item(SiteLocalStorage`1 storage, Object[] key)
 at System.Dynamic.UpdateDelegates.UpdateAndExecute2[T0,T1,TRet](CallSite site, T0 arg0, T1 arg1)
 at Microsoft.Scripting.Interpreter.DynamicInstruction`3.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.LightLambda.Run2[T0,T1,TRet](T0 arg0, T1 arg1)
 at IronPython.Runtime.Method.MethodBinding.SelfTarget(CallSite site, CodeContext context, Object target)
 at System.Dynamic.UpdateDelegates.UpdateAndExecute2[T0,T1,TRet](CallSite site, T0 arg0, T1 arg1)
 at Microsoft.Scripting.Interpreter.DynamicInstruction`3.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.LightLambda.Run4[T0,T1,T2,T3,TRet](T0 arg0, T1 arg1, T2 arg2, T3 arg3)
 at IronPython.Compiler.PythonCallTargets.OriginalCallTarget3(PythonFunction function, Object arg0, Object arg1, Object arg2)
 at Microsoft.Scripting.Interpreter.FuncCallInstruction`6.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.LightLambda.Run6[T0,T1,T2,T3,T4,T5,TRet](T0 arg0, T1 arg1, T2 arg2, T3 arg3, T4 arg4, T5 arg5)
 at System.Dynamic.UpdateDelegates.UpdateAndExecute5[T0,T1,T2,T3,T4,TRet](CallSite site, T0 arg0, T1 arg1, T2 arg2, T3 arg3, T4 arg4)
 at IronPython.Runtime.Types.LateBoundInitBinder.FastInitSite`2.CallTarget(CallSite site, CodeContext context, Object inst, T0 arg0, T1 arg1)
 at System.Dynamic.UpdateDelegates.UpdateAndExecute4[T0,T1,T2,T3,TRet](CallSite site, T0 arg0, T1 arg1, T2 arg2, T3 arg3)
 at IronPython.Runtime.Types.PythonType.FastTypeSite`2.CallTarget(CallSite site, CodeContext context, Object type, T0 arg0, T1 arg1)
 at System.Dynamic.UpdateDelegates.UpdateAndExecute4[T0,T1,T2,T3,TRet](CallSite site, T0 arg0, T1 arg1, T2 arg2, T3 arg3)
 at Microsoft.Scripting.Interpreter.DynamicInstruction`5.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)
 at Microsoft.Scripting.Interpreter.LightLambda.Run2[T0,T1,TRet](T0 arg0, T1 arg1)
 at IronPython.Compiler.PythonScriptCode.RunWorker(CodeContext ctx)
 at PyRevitBaseClasses.ScriptExecutor.ExecuteScript(PyRevitCommandRuntime& pyrvtCmd) 
```

Windows 7 home premium
pyRevit Version 4.6.14

pyRevit Environment: 

```
Microsoft Windows 7 [version 6.1.7601]
Executing User: D-Comp\Dacosta
Active User: D-Comp\Dacosta
Adming Access: No
%APPDATA% "C:\Users\Dacosta\AppData\Roaming"
Latest Installed .Net Framework: "4.7.2"
Installed .Net Target Packs: v3.5 v4.0 v4.5 v4.5.1 v4.5.2 v4.6 v4.6.1 v4.71 v4.....
pyRevit CLI 0.9.0.0
```

**Comment by yagbugs** [comment]
Hi Eirannejad, what would be a solution for this issue? Thank you in advance.

**Comment by eirannejad** [comment]
Yes there should be. I'll look at it this weekend.

**Comment by yagbugs** [comment]
Thank you Eirannejad.

**Comment by eirannejad** [comment]
Resolved and will be published with 4.6.15 this weekend.

Until then, overwrite the file below with the script inside this zip file: 
[script.zip](https://github.com/eirannejad/pyRevit/files/2846968/script.zip)

`extensions/pyRevitTools.extension/pyRevit.tab/Modify.panel/Patterns.splitpushbutton/Make Pattern.pushbutton/script.py`

**Comment by yagbugs** [comment]
Thank you Eirannejad. It's working now.
